tests-lit/tools/compilation_database.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/local/bin/python3
```

Comment:
This is exactly what the #629 is about. I would be fine with having this for now. And then discarding it when Mull starts to support that compilation database format.